src/Symfony/Component/Notifier/Tests/Transport/NullTransportFactoryTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Tests\Transport;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\NullTransport;
+use Symfony\Component\Notifier\Transport\NullTransportFactory;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
```

Comment:
You have to add `symfony/event-dispatcher-contracts` in `require-dev` section in `src/Symfony/Component/Notifier/composer.json` 

This is why tests fails